build_kafka.sh

```diff
@@ -29,18 +49,17 @@ mkdir -p build/etc/init
 mkdir -p build/etc/kafka
 mkdir -p build/var/log/kafka
 
-cp ${origdir}/kafka-broker.default build/etc/default/kafka-broker
-cp ${origdir}/kafka-broker.upstart.conf build/etc/init/kafka-broker.conf
-
 tar zxf ${origdir}/${src_package}
 cd kafka_${scala_version}-${version}
 mv config/log4j.properties config/server.properties ../build/etc/kafka
```

Comment:
Posting this here so I remember to ask but it's prob a salt thing... Is Kafka logging working okay for prod? I know there were issues in staging.

Reply:
I don't actually know.  I didn't remember any clear complaints.  :-/

Reply:
Ah yeah, it's potentially confusing. So I do see something in: https://staging.papertrailapp.com/groups/1/events?q=host%3Akafka%20-program%3A(cron%20rsyslog%20dhclient%20systemd%20snapteld%20clear-aoagent%20salt-minion) but I'm guessing that's not all of it as I can't seem to track down where the logs specified in `log4j.properties` are going:

```
log4j.appender.kafkaAppender.File=${kafka.logs.dir}/server.log
```

I assumed `kafka.logs.dir` = `/var/log/kafka`, but that's empty. If we can find out where it lives, we can monitor them with `remote_syslog2` or send them into syslog and rsyslog will forward 'em over.

Having said all that, perhaps there's just nothing to log and so no files existing is normal?